mantidimaging/gui/windows/live_viewer/view.py

```diff
@@ -36,9 +36,9 @@ def __init__(self, main_window: 'MainWindowView', live_dir_path: Path) -> None:
         # reposition to the right of the main window to be visible when launched from cli
         self.move(self.main_window.x() + self.main_window.width(), self.main_window.y())
 
-    def show_image(self, image: np.ndarray) -> None:
+    def show_most_recent_image(self, image: np.ndarray) -> None:
```

Comment:
Best to leave the name as `show_image()` because once the user can move the slider this will be used to show other images.